saleor/static/dashboard-next/hooks/useScroll.ts

```diff
@@ -0,0 +1,25 @@
+// tslint:disable:no-submodule-imports
+import * as throttle from "lodash/throttle";
```

Comment:
`import { throttle } from "lodash`;

Reply:
This imports whole `lodash` and I prefer to keep bundle as small as possible.